aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -201,6 +202,78 @@ module aptos_framework::jwks {
         assert!(num_bytes < MAX_FEDERATED_JWKS_SIZE_BYTES, error::invalid_argument(EFEDERATED_JWKS_TOO_LARGE));
     }
 
+    /// This can be called to install or update a set of JWKs for a federated OIDC provider.  This function should
+    /// be invoked to intially install a set of JWKs or to update a set of JWKs when a keypair is rotated.
+    ///
+    /// The `iss` parameter is the value of the `iss` claim on the JWTs that are to be verified by the JWK set.
+    /// `kid_vec`, `alg_vec`, `e_vec`, `n_vec` are String vectors of the JWK attributes `kid`, `alg`, `e` and `n` respectively.
+    ///
```

Comment:
I'm guessing this is all in the original documentation for `patch_federated_jwks`?  Though I don't see it.

What is `iss`?

* `alg` is the `algorithm`
* `kty` is the `key type`
* `kid` is `key id`?
* `e` is some cryptographic `e`
* `n` is some cryptographic `n`